### DISCUSSION TITLE
Option to disable input sorting

### DISCUSSION THREAD

**Discussion by pudsec** [projectdiscovery/dnsx#265] [Ideas]
The order of the input file is sorted ascending by dnsx, can there be an option to shuffle, or maybe not change the order of the input to begin with?

**Comment by MetzinAround** [comment]
@ehsandeep any possibilities on how the input file can be sorted differently?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
